### DISCUSSION TITLE
ESP Home version keeps changing back to 4.0 after power cycle

### DISCUSSION THREAD

**Discussion by DavidWicks** [Blackymas/NSPanel_HA_Blueprint#1982] [General]
I have had ESP versions past 4.0 running successfully, however after a HA update or after a power cycle of the NS Panel the ESPHome version always drops back to 4.0.  I then have to re-flash the panel and everything is working again without a problem.

I just tried it with 4.3.1 and have the same issue.

Any ideas ?

**Comment by edwardtfn** [comment]
- #1821
